packages/authgear-react-native/index.d.ts

```diff
@@ -85,36 +127,44 @@ export declare class OAuthError extends AuthgearError {
 }
 
 /**
+ * The path of the page in Authgear.
+ *
  * @public
  */
 export declare enum Page {
+  /**
+   * The path of the settings page in Authgear.
+   */
   Settings = "/settings",
+  /**
+   * The path of the indenties page in Authgear.
+   */
   Identities = "/settings/identities",
 }
 
 /**
- * PreAuthenticatedURLDeviceSecretNotFoundError
+ * PreAuthenticatedURLDeviceSecretNotFoundError occured when device secret is not found in the storage.
  *
  * @public
  */
 export declare class PreAuthenticatedURLDeviceSecretNotFoundError extends PreAuthenticatedURLNotAllowedError {}
 
 /**
- * PreAuthenticatedURLIDTokenNotFoundError
+ * PreAuthenticatedURLIDTokenNotFoundError occured when id token is not found in the storage.
  *
  * @public
  */
 export declare class PreAuthenticatedURLIDTokenNotFoundError extends PreAuthenticatedURLNotAllowedError {}
 
 /**
- * PreAuthenticatedURLInsufficientScopeError
+ * PreAuthenticatedURLInsufficientScopeError occured when the client does not have enough scopes for pre-authenticated URL.

```

Comment:
@louischan-oursky @tung2744 , when will this happen? missing sth in the config?

Reply:
> This may happen if the "Pre-authenticated URL" feature is not enabled when the user logged in this session. Ask the user to re-login to enable this feature.